hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/queries/AsNodeOperatorQueriesTest.java

```diff
@@ -0,0 +1,121 @@
+                    getAccountBalance(PAYER).exposingBalanceTo(currentPayerBalance::set),
+                    // perform paid query, pay for the query with payer account
+                    // the grpc client performs the query to different ports
+                    getAccountBalance(NODE_OPERATOR).payingWith(PAYER),
+                    sleepFor(3_000),
+                    getAccountBalance(PAYER).exposingBalanceTo(payerBalanceAfterFirstQuery::set),
+                    // perform free query to local port with asNodeOperator() method
+                    getAccountBalance(NODE_OPERATOR).payingWith(PAYER).asNodeOperator(),
+                    sleepFor(3_000),
+                    // assert payer account balance is not changed
+                    getAccountBalance(PAYER).exposingBalanceTo(payerBalanceAfterSecondQuery::set),
+                    doAdhoc(() -> {
+                        assertThat(currentPayerBalance.get())
+                                .withFailMessage("Balances are not equal!")
+                                .isEqualTo(payerBalanceAfterFirstQuery.get());
+                        assertThat(payerBalanceAfterFirstQuery.get())
+                                .withFailMessage("Balances are not equal!")
+                                .isEqualTo(payerBalanceAfterSecondQuery.get());
+                    }));
+        }
+
+        @HapiTest
+        final Stream<DynamicTest> nodeOperatorQueryVerifyPayerBalanceForAccountInfo_WithoutOpContext() {
+            // declare payer account balance variables
+            final AtomicLong currentPayerBalance = new AtomicLong();
+            final AtomicLong payerBalanceAfterFirstQuery = new AtomicLong();
+            final AtomicLong payerBalanceAfterSecondQuery = new AtomicLong();
+            return hapiTest(
+                    cryptoCreate(NODE_OPERATOR).balance(ONE_HUNDRED_HBARS),
+                    cryptoCreate(PAYER).balance(ONE_HUNDRED_HBARS),
+                    getAccountBalance(PAYER).exposingBalanceTo(currentPayerBalance::set),
+                    // perform paid query, pay for the query with payer account
+                    // the grpc client performs the query to different ports
+                    getAccountInfo(NODE_OPERATOR).payingWith(PAYER),
+                    sleepFor(3_000),
+                    getAccountBalance(PAYER).exposingBalanceTo(payerBalanceAfterFirstQuery::set),
+                    // perform free query to local port with asNodeOperator() method
+                    getAccountInfo(NODE_OPERATOR).payingWith(PAYER).asNodeOperator(),
+                    sleepFor(3_000),
+                    // assert payer account balance is not changed
+                    getAccountBalance(PAYER).exposingBalanceTo(payerBalanceAfterSecondQuery::set),
+                    doAdhoc(() -> {
+                        assertThat(payerBalanceAfterFirstQuery.get())
+                                .withFailMessage("Balances are equal!")
```

Comment:
```suggestion
                                // The balances after the first query should not be equal because the first query is not a node
                                // operator query, and should therefore be charged to `payer`
                                .withFailMessage("Balances are equal, but should not be!")
```

Likewise on the other assertion. It's helpful to somehow describe _why_ the failure is a problem

Reply:
Got it, I think I managed to upgrade the comments and assertion messages.